packages/dropdown-react/src/Dropdown.tsx

```diff
@@ -1,71 +1,69 @@
-                        {...getLabelProps()}
-                    >
-                        {label}
-                    </label>
-                    <input
-                        data-testid="jkl-dropdown-input"
-                        type="text"
-                        className={`jkl-dropdown__value ${inputValue ? "jkl-dropdown__value--show" : ""}`}
-                        value={inputValue !== null ? inputValue : undefined}
-                        readOnly
-                    />
-                    <ul className="jkl-dropdown__option" {...getMenuProps()}>
-                        {isOpen
-                            ? items.map((item, index) => (
-                                  <li
-                                      className={`jkl-dropdown__option--item ${
-                                          highlightedIndex === index ? "jkl-dropdown__option--item--highlighted" : ""
-                                      } ${inputValue === item ? "jkl-dropdown__option--item--selected" : ""}`}
-                                      {...getItemProps({
-                                          key: item,
-                                          index,
-                                          item,
-                                      })}
-                                  >
-                                      {item}
-                                  </li>
-                              ))
-                            : null}
-                    </ul>
-                </div>
-            )}
-        </Downshift>
-        /* eslint-enable jsx-a11y/label-has-associated-control, react/jsx-key */
+        <div className={className}>
+            <span className={`jkl-dropdown__label ${hasSelectedValue ? "jkl-dropdown__label--hover" : ""}`}>
+                {label}
+            </span>
+            <button className="jkl-dropdown__value">{hasSelectedValue ? selectedValue : "Velg"} </button>
+            <CoreToggle
+                className="jkl-dropdown__core-toggle"
+                popup={label}
+                hidden={hidden}
+                onToggle={() => setHidden(!hidden)}
+                onToggleSelect={(e: CoreToggleSelectEvent) => {
+                    e.target.hidden = true;
+                    e.target.button.focus();
+                    e.target.value = e.detail;
+                    setSelectedValue(e.detail.textContent);
+                }}
+            >
+                <ul className="jkl-dropdown__option-wrapper">
+                    {items.map((item) => {
+                        return (
+                            <li key={item}>
+                                <button className="jkl-dropdown__option">{item}</button>
+                            </li>
+                        );
+                    })}
+                </ul>
+            </CoreToggle>
+
+            <svg aria-hidden={"true"} className="jkl-dropdown__arrow" viewBox="0 0 100 100">
```

Comment:
I thought we had a chevron in css?

Reply:
Yes, but this is probably more lightweight, and just as configurable with `strokewidth` and `stroke`/`fill`. Worth a discussion at least! We should at least land on one of them.